tools/logger/logger.c

```diff
@@ -223,6 +223,11 @@ static void *wait_open(const char *watched_dir, const char *expected_file)
 
 	char * const fpath = malloc(strlen(watched_dir) + 1 + strlen(expected_file) + 1);
 
+	if (!fpath) {
+		fprintf(stdderr, "error: can't allocate memory\n");
```

Comment:
Typo: stderr

Reply:
Oops indeed, V2 uploaded. I had compile tested locally, or so I thought. Now went and retested all the four sof-logger PRs I pushed today.